src/librustc_typeck/collect.rs

```diff
@@ -1958,7 +1958,7 @@ pub fn compute_bounds<'gcx: 'tcx, 'tcx>(astconv: &AstConv<'gcx, 'tcx>,
 
     Bounds {
         region_bounds: region_bounds,
-        builtin_bounds: builtin_bounds,
+        auto_traits: auto_traits,
```

Comment:
I'm not sure this partitioning should even exist until a `TyDynamic` is created.